static/js/pages/independent-expenditures.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+var tables = require('../modules/tables');
+var helpers = require('../modules/helpers');
+var columns = require('../modules/columns');
+
+var FilterPanel = require('fec-style/js/filter-panel').FilterPanel;
+var filterTags = require('fec-style/js/filter-tags');
+
+var expenditureTemplate = require('../../templates/independent-expenditures.hbs');
+
+var columns = [
+  {
+    data: 'committee',
+    orderable: false,
+    className: 'all',
+    render: function(data, type, row, meta) {
```

Comment:
'meta' is defined but never used.<br>'row' is defined but never used.<br>'type' is defined but never used.